src/views/preview/project-view.jsx

```diff
@@ -255,6 +256,10 @@ class Preview extends React.Component {
                 false // Do not show cloud/username alerts again
             );
         }
+
+        if (!prevProps.user.id && this.props.user.id) {
+            sendUserProperties(this.props.user);
+        }
```

Comment:
Should these be sent over only if the user is allowed to participate in the onboarding project, i.e. is part of either the A or B group?

Reply:
Yeah, I have implemented the check in `sendUserProperties` in the UEPR-71 ticket as we include the toggling logic